extensions/wikia/SiteWideMessages/SpecialSiteWideMessages_body.php

```diff
@@ -300,6 +300,13 @@ function execute($subpage) {
 	}
 
 	//DB functions
+
+	/**
+	 * @param User $mSender
+	 * @param $mText
+	 * @param $formData
+	 * @return array
+	 */
 	private function sendMessage( $mSender, $mText, $formData ) {
 		global $wgExternalSharedDB, $wgSpecialsDB, $wgUser;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this code to rely on RequestContext instead of context-dependent globals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3AContextualWgVariableUsage)

[project-id]: wikia:app